File: javascript/json-transform/src/functions/TransformerFunctionRaw.ts
import TransformerFunction from "./common/TransformerFunction";
import FunctionContext from "./common/FunctionContext";

class TransformerFunctionRaw extends TransformerFunction {
  constructor() {
    super({
      inputIsRaw: true,
    });
  }

  override async apply(context: FunctionContext): Promise<any> {
    return context.getJsonElement(null, false);
  }
}

export default TransformerFunctionRaw;

File: java/json-transform/src/main/java/ai/bizone/jsontransform/functions/TransformerFunctionRaw.java
package ai.bizone.jsontransform.functions;

import ai.bizone.jsontransform.functions.common.FunctionContext;
import ai.bizone.jsontransform.functions.common.TransformerFunction;

public class TransformerFunctionRaw extends TransformerFunction {
    public TransformerFunctionRaw() {
        super();
    }
    @Override
    public Object apply(FunctionContext context) {
        return context.getJsonElement(null, false);
    }
}
